src/components/DeviceStatusGauge.js

```diff
@@ -0,0 +1,241 @@
+import { h } from 'preact'
+import _ from 'lodash'
+import Pie from 'paths-js/pie'
+import styled, { withTheme } from 'styled-components'
+import { compose } from 'recompose'
+import ResinDeviceStatus from 'resin-device-status'
+
+import hoc from '../hoc'
+import theme from '../theme'
+
+const { colors, font } = theme
```

Comment:
We should be getting the theme from `.props`.

Reply:
agreed